src/main/java/bio/terra/service/profile/flight/update/ProfileUpdateFlight.java

```diff
@@ -18,16 +18,13 @@ public ProfileUpdateFlight(FlightMap inputParameters, Object applicationContext)
     ProfileService profileService = appContext.getBean(ProfileService.class);
     GoogleProjectService googleProjectService = appContext.getBean(GoogleProjectService.class);
 
-    BillingProfileUpdateModel request =
-        inputParameters.get(
-            JobMapKeys.REQUEST.getKeyName(), bio.terra.model.BillingProfileUpdateModel.class);
+    BillingProfileUpdateModel request = JobMapKeys.REQUEST.get(inputParameters);
 
-    AuthenticatedUserRequest user =
-        inputParameters.get(JobMapKeys.AUTH_USER_INFO.getKeyName(), AuthenticatedUserRequest.class);
+    AuthenticatedUserRequest user = JobMapKeys.AUTH_USER_INFO.get(inputParameters);
 
-    addStep(new UpdateProfileRetrieveExistingProfileStep(profileService, request, user));
+    addStep(new UpdateProfileRetrieveExistingProfileStep(profileService, request));
```

Comment:
What happened to the user?

Reply:
It turned out the step never used it. This was an incidental change I made while looking at the code and isn't part of the map key change.